src/Theme.php

```diff
@@ -219,6 +226,42 @@ public function getParent(): ?Theme
         return $this->parent;
     }
 
+
+    /**
+     * Set theme screenshot.
+     */
+
+    public function setScreenshot(string $screenshot): self
+    {
+
+        $this->screenshot = $screenshot;
+
+        return $this;
+    }
+
+    public function getScreenshotName(): string|null
+    {
+        return $this->screenshot;
+    }
+
+    public function getScreenshotImageUrl(): string
+    {
+        $screenshotImagePath = $this->getAssetsPath($this->screenshot);
```

Comment:
To get the URL you should use `$this->url($this->screenshot);`  otherwise you get the absolute path of the image file.